2024/FOSDEM/osf_on_amd_5th.md

```diff
@@ -0,0 +1,287 @@
+
+---
+
+# AMD server status - last year vs now
+
+.center[.image-30[![](/img/EpycProcessor.jpg)]]
+
+.center.footnote[
+[AMD EPYC photo](https://upload.wikimedia.org/wikipedia/commons/7/7d/EpycProcessor.jpg)
+by Raysonho @ Open Grid Scheduler / Grid Engine, CC0, via Wikimedia Commons
+]
+
+.left-column50[
+
+### Early 2023
+
+- Some new initiative from Oxide on OSF for AMD Milan (EPYC 7002 series)
+  server platform presented at [OSFC
+  2022](https://www.osfc.io/2022/talks/i-have-come-to-bury-the-bios-not-to-open-it-the-need-for-holistic-systems/)
+- Nothing new on official OSF support on servers from AMD
+]
+
+.right-column50[
+
+### Now
+
+- New approach to open-source firmware on AMD server - OpenSIL
+- Porting AGESA to AMD FSP and maintaining it was too costly
+- OpenSIL announced on [OCP Regional Summit 2023
+  Prague](https://www.youtube.com/watch?v=q_y6Y1JTq0I)
+]
+
+---
+
+# AMD server status - OpenSIL
+
+- OpenSIL - open-source Silicon Initialization Library
+    * Scalable with any host firmware interface/framework
+    * Proof of Concept code for Genoa-based (AMD EPYC 9004 processors)
+      reference board Onyx available on
+      [GitHub](https://github.com/openSIL/openSIL)
+- [coreboot source also available and merged to upstream
+  repository](https://review.coreboot.org/q/topic:%22amd_genoa_opensil%22)
+- UEFI EDK2-based PoC code also available on GitHub:
+    * [opensil-uefi-interface](https://github.com/openSIL/opensil-uefi-interface)
+    * [EDK2 Platforms](https://github.com/openSIL/EDKII-Platform)
+- More about OpenSIL:
+    * [OCP Global Summit 2023](https://www.youtube.com/watch?v=nSVKKMkcIPE)
+    * [OSFC 2023](https://vimeo.com/878219919)
+
+---
+
+# AMD OpenSIL future
+
+.left-column55[
+.center[.image-90[![](/img/opensil_github.png)]]
+]
+
+.right-column45[
+- The OpenSIL is constantly developed and improved by AMD and partners
+- Planned to go **production in 2026** with the 6th generation of AMD EPYC
+  processors
+- AMD plans to cover the **client segment with OpenSIL support** too (Ryzen
+  desktop and mobile processors)
+]
+
+---
+
+# PC Engines platforms - yet another chance?
+
+- Last year we announced the end of PC Engines' sponsorship of open-source
+  firmware
+- We tried to gather community interested in the open-source firmware for apu2
+  board and launch a subscription model, however the response was very little
+  and we didn't succeed
+- This year we see another chance to revive the project under
+  [Dasharo](https://dasharo.com) brand
+
+.center.image-40[![](/img/pcengines_logo.png)]
+.center[
+.image-30[![](/img/apu4d4.png)]
+.image-20[![](/img/dasharo-sygnet.png)]
+]
+
+???
+
+---
+
+# Dasharo
+
+- In [Dasharo](https://dasharo.com) we carefully select hardware platforms,
+  which are quite popular
+    * Of course most such boards will be community-driven effort and the work
+    would be done in free time thus the releases may not be that frequent
```

Comment:
This is described more extensively in FAQ: https://docs.dasharo.com/osf-trivia-list/dasharo/#can-you-port-dasharo-to-my-mainboard

Reply:
I gave up on this bulletpoint